<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Start_Button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="energy_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>energy</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>time</string>
    </property>
   </widget>
   <widget class="QLabel" name="energy_val">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_val">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="pulse_0_val">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="pulse_1_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>pulse y</string>
    </property>
   </widget>
   <widget class="QLabel" name="count_label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>count</string>
    </property>
   </widget>
   <widget class="QLabel" name="pulse_0_label">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>pulse x</string>
    </property>
   </widget>
   <widget class="QLabel" name="pulse_1_val">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="count_val">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>10</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="mass_val">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
   <widget class="QLabel" name="mass_label">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>mass</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
